pages/data/index.vue

```diff
@@ -177,15 +177,16 @@ const searchData = {
 }
 
 import createClient from '@/plugins/contentful.js'
-import SearchMapPopup from "../../components/SearchMapPopup/SearchMapPopup"
+import SearchMapPopup from "../../components/SearchMapPopup/SearchMapPopup";
```

Comment:
When importing we should use the `@` alias in the import lines, like we are doing for `createClient`